File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        temp=list(Counter(nums))
        print(temp)
        
        
        j=0
        for i in range(len(nums)):
            if j<len(temp):
                nums[i]=temp[j]
                j+=1
            else:
                nums[i]=0
        
        return len(temp)

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int count=1,pos=1;
        bool updated=true;
        for(int i=1;i<nums.size();i++){
            if(nums[i]!=nums[i-1]){
                count+=1;
                
                nums[pos]=nums[i];
                pos+=1;
                updated=true;
            }else{
                if(updated){
                    // pos=1;
                    updated=false; 
                }
                
            }
        }
        return count;
    }
};